### PR TITLE
docs(core): clean-up outdated content from docs and CNW readme

### PR DESCRIPTION
**Pull request by jaysoo**
This PR:

1. Removes the `{{getting-started}}` template string from [CNW README.md](https://www.npmjs.com/package/create-nx-workspace), which was removed before from generation but this file missed an update.
2. Updates docs for `run-scripts` to show `project.json` examples, not `workspace.json` (we link to this from PDV and it's very awkward).
3. Removes unused run-script and run-commands example files from `packages/workspace/docs`. These were moved to `packages/nx/docs` but somehow the old files remained even though they aren't used.
4. Updates host/remotes React generators to mention `nx.json` not `workspace.json`. The option is already renamed `--skipNxJson` but the description does not match.

## Related Issue(s)

Fixes #

### CODE REVIEW

**Review by isaacplmann — APPROVED** [review]

File: docs/generated/packages/react/generators/host.json [hunk 1]
```diff
@@ -84,7 +84,7 @@
         "x-priority": "internal"
       },
       "skipNxJson": {
-        "description": "Skip updating workspace.json with default options based on values provided to this app (e.g. babel, style).",
+        "description": "skip updating nx.json with default options based on values provided to this app (e.g. babel, style).",
```
Comment (Coly010) [inline docs/generated/packages/react/generators/host.json:87]: Are we intentionally lower casing the first word here? 
Other options in the schema are title cased for the first word.

File: docs/generated/packages/react/generators/remote.json [hunk 2]
```diff
@@ -90,7 +90,7 @@
         "x-priority": "internal"
       },
       "skipNxJson": {
-        "description": "Skip updating workspace.json with default options based on values provided to this app (e.g. babel, style).",
+        "description": "skip updating nx.json with default options based on values provided to this app (e.g. babel, style).",
```
Comment (Coly010) [inline docs/generated/packages/react/generators/remote.json:93]: Same as above

File: packages/react/src/generators/host/schema.json [hunk 3]
```diff
@@ -90,7 +90,7 @@
       "x-priority": "internal"
     },
     "skipNxJson": {
-      "description": "Skip updating workspace.json with default options based on values provided to this app (e.g. babel, style).",
+      "description": "skip updating nx.json with default options based on values provided to this app (e.g. babel, style).",
```
Comment (Coly010) [inline packages/react/src/generators/host/schema.json:93]: Same as above

File: packages/react/src/generators/remote/schema.json [hunk 4]
```diff
@@ -96,7 +96,7 @@
       "x-priority": "internal"
     },
     "skipNxJson": {
-      "description": "Skip updating workspace.json with default options based on values provided to this app (e.g. babel, style).",
+      "description": "skip updating nx.json with default options based on values provided to this app (e.g. babel, style).",
```
Comment (Coly010) [inline packages/react/src/generators/remote/schema.json:99]: Same as above

**Review by Coly010 — APPROVED** [review]
Approved, but some comments

**Pull request merged**